test/new_relic/agent/obfuscator_test.rb

```diff
@@ -48,12 +48,12 @@ def test_decoding_blank
 
   def test_decoding_empty_key
     obfuscator = NewRelic::Agent::Obfuscator.new("")
-    assert_equal "querty", obfuscator.encode('querty')
+    assert_equal "query", obfuscator.encode('query')
```

Comment:
Or `qwerty`? but that didn't seem likely...

Reply:
I think the input doesn't particularly matter for this test. The change seems fine.